patches/react-native-plaid-link-sdk+10.0.0.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/react-native-plaid-link-sdk/android/build.gradle b/node_modules/react-native-plaid-link-sdk/android/build.gradle
+index 5fa85a5..0b28564 100644
+--- a/node_modules/react-native-plaid-link-sdk/android/build.gradle
++++ b/node_modules/react-native-plaid-link-sdk/android/build.gradle
```

Comment:
@mczernek did this that patch after bumping up kotlin version, so probably it's because there were some issues. He left Expensify team in SWM already, but I will catch him and ensure my guesses are correct.

Reply:
Ok thanks for the explanation. If I run `npm i` on this branch though, I'm getting this error:

```
**ERROR** Failed to apply patch for package react-native-plaid-link-sdk at path

[...]

  Info:
    Patch file: patches/react-native-plaid-link-sdk+10.0.0.patch
    Patch was made for version: 10.0.0
    Installed version: 10.8.0
```


I imagine it's because we bumped that lib's required version from 10.0.0 to 10.8.0 while the PR was being worked on. I looked at `node_modules/react-native-plaid-link-sdk/android/build.gradle` for 10.8.0 and it doesn't contain `apply plugin: "kotlin-android-extensions"`, so _maybe_ this patch isn't needed any longer with `react-native-plaid-link-sdk` 10.8.0?